CONTRIBUTE.md

```diff
@@ -0,0 +1,102 @@
+# Contributing Guidelines for firecamp.io
```

Comment:
The file name should be `CONTRIBUTING.md` as per the [guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors)

you can also point the related issue here: https://github.com/FirecampDev/Firecamp/issues/9